js/memory.js

```diff
@@ -1,9 +1,27 @@
 class MemoryGame {
-  constructor(cards){
+  constructor(cards) {
     this.cards = cards;
-    // add the rest of the class properties here
+    // this.pickedCards = pickedCards;
+    // this.pairsClicked = pairsClicked;
+    // this.pairsGuessed = pairsGuessed;
+  }
+  shuffleCards(cardsObj) {
+    if (!cardsObj) {
+      return undefined;
+    }
+    for (let i = 0; i < cardsObj.length; i++) {
+      let rand = Math.floor(Math.random() * cardsObj.length);
+      cardsObj.push(cardsObj[rand]);
+      cardsObj.splice(rand, 1);
+    }
```

Comment:
I see what you do here, and it makes sense.
Would there be an alternative way that ensures that every card is shuffled, perhaps by making use of the index iterated through?